src/styles/_variables.scss

```
@@ -0,0 +1,61 @@
+$theme-colors: (
+    "mainColor": #155bd5,
+    "secondColor": white,
+    "textColor": #6d6d6d,
```

We already knew context for vars. It's color.

```suggestion
    "main": #155bd5,
    "secondary": white,
    "text": #6d6d6d,
```